clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/ImmutableChangeCapturePubSubMessage.java

```diff
@@ -108,6 +138,11 @@ public java.nio.ByteBuffer getReplicationMetadataPayload() {
     return replicationMetadataPayload;
   }
 
+  @Override
```

Comment:
NIT: you can also remove this `@Override` now since it's no longer in the interface.